Error when building - SohoDataGridCellChangeEvent is not generic

**Describe the bug**
After updating to Enterprise-NG v18.1.0 or v18.2.0  I am running into this error when I run ng build:
Error: node_modules/ids-enterprise-ng/lib/datagrid/soho-datagrid.component.d.ts:516:30 - error TS2315: Type 'SohoDataGridCellChangeEvent' is not generic. 

I changed line 516 in soho-datagrid.component.d.ts from:
`cellchange: EventEmitter<SohoDataGridCellChangeEvent<Object>>;`
 
to:
`cellchange: EventEmitter<SohoDataGridCellChangeEvent>;`
 
and it is now building without any issues. This is with ids-enterprise-ng v18.2.1.

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -1,8 +1,8 @@
 # What's New with Enterprise-NG
 
-## 18.2.0
+## 18.3.0
 
-## 18.2.0 Fixes
+## 18.3.0 Fixes
 
 - `[Datagrid]` Removed cellchange Generic Type. ([#1757](https://github.com/infor-design/enterprise-ng/issues/1757))
 

--- a/docs/CHANGELOG.md
+++ b/docs/CHANGELOG.md
@@ -2,6 +2,12 @@
 
 ## 18.2.0
 
+## 18.2.0 Fixes
+
+- `[Datagrid]` Removed cellchange Generic Type. ([#1757](https://github.com/infor-design/enterprise-ng/issues/1757))
+
+## 18.2.0
+
 ## 18.2.0 Features
 
 - `[General]` Added 4.98.0 version

--- a/projects/ids-enterprise-typings/lib/datagrid/soho-datagrid.d.ts
+++ b/projects/ids-enterprise-typings/lib/datagrid/soho-datagrid.d.ts
@@ -1435,9 +1435,9 @@ interface SohoDataGridSelectedEvent {
   rowData?: SohoDataGridSelectedRow[] | SohoDataGridSelectedRow;
 }
 
-interface SohoDataGridCellChangeEvent<T = Object> {
+interface SohoDataGridCellChangeEvent {
   api?: any;
-  rowData?: T;
+  rowData?: any;
   row?: number;
   relativeRow?: number;
   cell?: number;

--- a/src/app/datagrid/datagrid-custom-component.demo.ts
+++ b/src/app/datagrid/datagrid-custom-component.demo.ts
@@ -3,149 +3,150 @@ import {
 } from 'rxjs';
 
 import {
-    Component,
-    ViewChild,
-    OnInit,
-    Inject,
+  Component,
+  ViewChild,
+  OnInit,
+  Inject,
 } from '@angular/core';
 import { SohoDataGridComponent } from 'ids-enterprise-ng';
 
 @Component({
-    selector: 'app-formatter',
-    template: '<p style="color: red">formatted</p>',
+  selector: 'app-formatter',
+  template: '<p style="color: red">formatted</p>',
 })
 export class FormatterComponent {
-    public data: any;
+  public data: any;
 
-    constructor(@Inject('args') public args: SohoDataGridPostRenderCellArgs) {
-        this.data = args.value;
-        console.log('component formatter ran', this.data);
-    }
+  constructor(@Inject('args') public args: SohoDataGridPostRenderCellArgs) {
+    this.data = args.value;
+    console.log('component formatter ran', this.data);
+  }
 }
 
 @Component({
-    selector: 'app-datagrid-custom-demo',
-    templateUrl: 'datagrid-custom-component.demo.html',
-    providers: []
+  selector: 'app-datagrid-custom-demo',
+  templateUrl: 'datagrid-custom-component.demo.html',
+  providers: []
 })
 export class DataGridCustomComponent implements OnInit {
-    @ViewChild(SohoDataGridComponent)
-    sohoDataGridComponent?: SohoDataGridComponent;
-    gridOptions?: SohoDataGridOptions = undefined;
-    columns!: any[];
-    pagingData: any[] = [];
+  @ViewChild(SohoDataGridComponent)
+  sohoDataGridComponent?: SohoDataGridComponent;
+  gridOptions?: SohoDataGridOptions = undefined;
+  columns!: any[];
+  pagingData: any[] = [];
 
-    constructor() { }
-    ngOnInit() {
-        this.pagingData = [
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building10',
-                orderDate: '10.10.2022',
-                buyer: 'Company1',
-                spoStatus: 'Approved',
-            },
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building1',
-                orderDate: '10.10.2022',
-                buyer: 'Company2',
-                spoStatus: 'Declined',
-            },
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building2',
-                orderDate: '10.10.2022',
-                buyer: 'Company3',
-                spoStatus: 'Approved',
-            },
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building0011',
-                orderDate: '10.10.2022',
-                buyer: 'Company4',
-                spoStatus: 'Declined',
-            },
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building00009',
-                orderDate: '10.10.2022',
-                buyer: 'Company5',
-                spoStatus: 'Incomplete',
-            },
-            {
-                spo: 'PRJ0000911100',
-                project: 'Building003',
-                orderDate: '10.10.2022',
-                buyer: 'Company6',
-                spoStatus: 'Incomplete',
-            },
-        ];
-        this.columns = [
-            {
-                id: 'spo',
-                name: 'SPO',
-                field: 'spo',
-                editor: Soho.Editors.Input,
-                sortable: true,
-                filterType: 'text',
-            },
-            {
-                id: 'project',
-                name: 'Project',
-                field: 'project',
-                sortable: true,
-                component: FormatterComponent,
-                componentInputs: {},
-                postRender: true,
-                filterType: 'text',
-            },
-            {
-                id: 'orderDate',
-                name: 'Order Date',
-                field: 'orderDate',
-                formatter: Soho.Formatters.Date,
-                sortable: true,
-                filterType: 'date',
-                dateFormat: 'MM.dd.yyyy',
-            },
-            {
-                id: 'spoStatus',
-                name: 'SPO Status',
-                field: 'spoStatus',
-                sortable: true,
-                formatter: Soho.Formatters.Input,
-                filterType: 'text',
-            },
-            {
-                id: 'buyer',
-                name: 'Buyer',
-                field: 'buyer',
-                formatter: Soho.Formatters.Input,
-                editor: Soho.Editors.Input,
-                filterType: 'text',
-                sortable: true,
-            },
-        ];
+  constructor() { }
+  ngOnInit() {
+    this.pagingData = [
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building10',
+        orderDate: '10.10.2022',
+        buyer: 'Company1',
+        spoStatus: 'Approved',
+      },
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building1',
+        orderDate: '10.10.2022',
+        buyer: 'Company2',
+        spoStatus: 'Declined',
+      },
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building2',
+        orderDate: '10.10.2022',
+        buyer: 'Company3',
+        spoStatus: 'Approved',
+      },
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building0011',
+        orderDate: '10.10.2022',
+        buyer: 'Company4',
+        spoStatus: 'Declined',
+      },
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building00009',
+        orderDate: '10.10.2022',
+        buyer: 'Company5',
+        spoStatus: 'Incomplete',
+      },
+      {
+        spo: 'PRJ0000911100',
+        project: 'Building003',
+        orderDate: '10.10.2022',
+        buyer: 'Company6',
+        spoStatus: 'Incomplete',
+      },
+    ];
+    this.columns = [
+      {
+        id: 'spo',
+        name: 'SPO',
+        field: 'spo',
+        editor: Soho.Editors.Input,
+        sortable: true,
+        filterType: 'text',
+      },
+      {
+        id: 'project',
+        name: 'Project',
+        field: 'project',
+        sortable: true,
+        component: FormatterComponent,
+        componentInputs: {},
+        postRender: true,
+        filterType: 'text',
+      },
+      {
+        id: 'orderDate',
+        name: 'Order Date',
+        field: 'orderDate',
+        formatter: Soho.Formatters.Date,
+        sortable: true,
+        filterType: 'date',
+        dateFormat: 'MM.dd.yyyy',
+      },
+      {
+        id: 'spoStatus',
+        name: 'SPO Status',
+        field: 'spoStatus',
+        sortable: true,
+        formatter: Soho.Formatters.Input,
+        filterType: 'text',
+      },
+      {
+        id: 'buyer',
+        name: 'Buyer',
+        field: 'buyer',
+        formatter: Soho.Formatters.Input,
+        editor: Soho.Editors.Input,
+        filterType: 'text',
+        sortable: true,
+      },
+    ];
 
-        this.gridOptions = {
-            columns: this.columns,
-            dataset: this.pagingData,
-            selectable: 'single',
-            showNewRowIndicator: false,
-            editable: true,
-            filterable: true,
-            clickToSelect: false,
-            rowTemplate: `<div class="datagrid-cell-layout"></div>`,
-            toolbar: {
-                keywordFilter: false,
-            },
-        };
+    this.gridOptions = {
+      columns: this.columns,
+      dataset: this.pagingData,
+      selectable: 'single',
+      showNewRowIndicator: false,
+      editable: true,
+      filterable: true,
+      clickToSelect: false,
+      rowTemplate: `<div class="datagrid-cell-layout"></div>`,
+      toolbar: {
+        keywordFilter: false,
+      },
+    };
+  }
+
+  onCellChange(event: any) {
+    if (this.sohoDataGridComponent) {
+      this.sohoDataGridComponent.updateRow(event.row, event.rowData);
     }
-
-    onCellChange(event: any) {
-        if (this.sohoDataGridComponent) {
-            this.sohoDataGridComponent.updateRow(event.row, event.rowData);
-        }
-    }
+    console.log('onCellChange', event);
+  }
 }

